src/renderers/dom/shared/ReactDOMComponent.js

```diff
@@ -862,6 +864,11 @@ ReactDOMComponent.Mixin = {
           DOMProperty.properties[propKey] ||
           DOMProperty.isCustomAttribute(propKey)) {
         DOMPropertyOperations.deleteValueForProperty(getNode(this), propKey);
+      } else if (this._namespaceURI === DOMNamespaces.svg) {
```

Comment:
Did you mean for this to be above the previous case? I think it works out the same either way, so I guess it doesn't really matter.